### PR TITLE
Implement calculate standard deviation

### PR DESCRIPTION
**Pull request by PANDASANG1231**

### CODE REVIEW

File: R/get_matching_column_names.R [hunk 1]
```diff
@@ -13,6 +13,7 @@
 #' data <- data.frame(week_payment1 = c(1, 2, 3), week_payment2 = c(1, 2, 3),
 #'                    week_payment3 = c(1, 2, 3), othercolumn = c(5, 6, 7))
 #' get_matching_column_names(data, "week_payment")
+library(stringr)
```
Comment (NikitaShymberg) [inline R/get_matching_column_names.R]: Do we need this line?

File: R/calculate_standard_deviation.R [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  colnames <- get_matching_column_names(df, pattern)
+
+
+  # check if there are matching column names
+  if(length(colnames) == 0){
+    stop("No column names found matching pattern")
+  }
+
+  # calculate average of colnames
+  rst <- apply(df[, colnames, drop = F], 1, sd)
```
Comment (NikitaShymberg) [inline R/calculate_standard_deviation.R:41]: When I run `check()` I get this error:

```
 calculate_percentage_change: no visible binding for global variable
    ‘None’
  calculate_standard_deviation: no visible binding for global variable
    ‘sd’
  Undefined global functions or variables:
    None sd
  Consider adding
    importFrom("stats", "sd")
  to your NAMESPACE file.
```

Does it pass on your end?

**Review by NikitaShymberg — CHANGES_REQUESTED** [review]

File: R/calculate_standard_deviation.R (hunk 2 above)
Comment (NikitaShymberg) [reply R/calculate_standard_deviation.R:41]: I notice that the error about the `None` comes from the definition of `calculate_percentage_change` I _think_ this has been fixed in main.

Comment (PANDASANG1231) [reply R/calculate_standard_deviation.R:41]: Sorry, I was outside this afternoon. Glad that it is fixed

File: R/get_matching_column_names.R (hunk 1 above)
Comment (PANDASANG1231) [reply R/get_matching_column_names.R]: I just deleted this line

**Review by SonQBChau — APPROVED** [review]

**Review by SonQBChau — APPROVED** [review]

**Review by NikitaShymberg — APPROVED** [review]

**Pull request merged**